matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/clients/rest/dtos/events/RunnersRequest.java

```diff
@@ -43,4 +44,12 @@ public void setIncludeWithdrawn(boolean includeWithdrawn) {
         this.includeWithdrawn = includeWithdrawn;
     }
 
+    public boolean isIncludePrices() {
+        return includePrices;
+    }
+
+    public void setIncludePrices(boolean includePrices) {
```

Comment:
what about to use builder pattern for requests?

Reply:
Agreed. I'll use the builder pattern for every request :+1: